docs/reference/esql/functions/aggregation-functions.asciidoc

```diff
@@ -17,7 +17,7 @@ The <<esql-stats-by>> command supports these aggregate functions:
 * <<esql-min>>
 * <<esql-agg-percentile>>
 * experimental:[] <<esql-agg-st-centroid>>
-* <<esql-agg-sum>>
```

Comment:
I believe we use this identifier as the URL for the page for the aggregation's docs. This'll shift the url. I suppose that's ok. Not great, but we can live it with it at this early point.

Reply:
That's right! I thought we made a page per agg. If it's just the anchor that's ok. Not as good as not breaking it, but it's not a problem.